k8s/commander.sh

```diff
@@ -17,8 +17,16 @@ function deploy {
     DEPLOY_SECRETS=${5:-NO}
     K8S_NAMESPACE="sumo-${REGION_ENV}"
 
-    export KUBECTL_BIN="./regions/${REGION}/kubectl"
-    export KUBECONFIG="./regions/${REGION}/kubeconfig"
+    if [ -f "./regions/${REGION}/kubectl" ] ; then
+        KUBECTL_BIN="./regions/${REGION}/kubectl"
+    else
+        KUBECTL_BIN=$(which kubectl)
```

Comment:
Shouldn't  we have a requirement to use a kubectl version with an offset of 1 from the apiserver?

Reply:
Do you want me to add the logic of checking the versions in the script, or document the requirement?

Reply:
Documenting the requirement somewhere in this readme should be enough

Reply:
documented here https://github.com/mozilla/kitsune/pull/4552/files#diff-1d8ece659445546a89a71faddc37a9b3fca6fc8e8870b3bb40cadb707e54e946R51